cirq-ionq/cirq_ionq/ionq_client.py

```diff
@@ -301,11 +301,17 @@ def _make_request(
                         'IonQ could not find requested resource.'
                     )
                 if not _is_retriable(response.status_code):
+                    error = {}
+                    try:
+                        error = response.json()
+                    except json.decoder.JSONDecodeError:
```

Comment:
I think this will reference `json` from the parameter.  Probably best solution is to `import json.decoder as jd` or something like that and then us that reference here.

Reply:
Thinking about this more, I probably shouldn't coverage ignore the except case if I already did a bug